mlfinlab/labeling/tail_sets.py

```diff
@@ -8,29 +8,40 @@ class TailSetLabels:
     Tail set labels are a classification labeling technique introduced in the following paper: Nonlinear support vector
     machines can systematically identify stocks with high and low future returns. Algorithmic Finance, 2(1), pp.45-58.
 
-    A tail set is defined to be a group of stocks whose volatility-adjusted price change is in the highest or lowest
+    A tail set is defined to be a group of stocks whose volatility-adjusted return is in the highest or lowest
     quantile, for example the highest or lowest 5%.
 
     A classification model is then fit using these labels to determine which stocks to buy and sell in a long / short
     portfolio.
     """
 
-    def __init__(self, prices, window, mean_abs_dev):
+    def __init__(self, prices, n_bins, vol_adj=None, window=None):
         """
         :param prices: (pd.DataFrame) Asset prices.
-        :param window: (int) Window period used in the calculation of the volatility.
-        :param mean_abs_dev: (bool) To use the mean absolute deviation or traditional standard deviation.
+        :param n_bins: (int) Number of bins to determine the quantiles for defining the tail sets. The top and
+                        bottom quantiles are considered to be the positive and negative tail sets, respectively.
+        :param vol_adj: (bool) Whether to take volatility adjusted returns. Allowable inputs are ``None``,
+                        ``mean_abs_dev``, and ``stdev``.
+        :param window: (int) Window period used in the calculation of the volatility adjusted returns, if vol_adj is not
+                        None. Has no impact if vol_adj is None.
         """
+        assert prices.shape[1] > n_bins, "n_bins exceeds the number of stocks!"
```

Comment:
This will also trigger an error if `prices.shape[1] == n_bins` - when each stock has a separate bin.